doc/source/data/creating-datasets.rst

```diff
@@ -304,6 +304,26 @@ are supported for each of these storage systems.
     :start-after: __read_parquet_az_begin__
     :end-before: __read_parquet_az_end__
 
+.. tabbed:: SQL Databases
+
+  Read data from databases that provide a Python DB API2-compliant connector.
+
+  .. testcode::
+
+    import sqlite3
+
+    import ray
+
+    # Create a simple database
+    connection = sqlite3.connect("example.db")
+    connection.execute("CREATE TABLE movie(title, year, score)")
```

Comment:
Make this example less-fake, can we write some rows into the table first? Reference - https://www.sqlitetutorial.net/sqlite-insert/ .

```
INSERT INTO table1 (column1,column2 ,..)
VALUES 
   (value1,value2 ,...),
   (value1,value2 ,...),
    ...
   (value1,value2 ,...);
```